packages/clay-toolbar/src/Link.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayLink from '@clayui/link';
+import classNames from 'classnames';
+import React from 'react';
+
+export interface IProps extends React.ComponentProps<typeof ClayLink> {
+	/**
+	 * FLag that determines if the Link will have a `component-link` class, allowing CSS overrides.
+	 */
+	componentLink?: boolean;
```

Comment:
Same for this, I think we can just remove these as configurable and always have this class on the component. If it becomes and issue later we can always add the API in.

Its easier to add an API than remove one.